docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+
+version: '3'
+
+services:
+  web:
+    build: .
+    # command: bash -c "python project/manage.py makemigrations && python project/manage.py migrate && python project/manage.py runserver 0.0.0.0:8001"

```

Comment:
Ler's remove commented code, as a general practice.

Reply:
Oh, sorry. My bad leaving it there.

Reply:
Fixed it.